security-openid-connect-multi-tenancy-quickstart/pom.xml

```
@@ -13,7 +13,7 @@
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
         <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
-        <keycloak.version>13.0.0</keycloak.version>
+        <keycloak.version>17.0.0</keycloak.version>
```

My preference is to be explicit in version definition
```suggestion
        <keycloak.version>17.0.0-legacy</keycloak.version>
```